src/app.html

```diff
@@ -6,7 +6,7 @@
 		<meta name="viewport" content="width=device-width, initial-scale=1" />
 		%sveltekit.head%
 	</head>
-	<body class="h-full">
+	<body class="h-full bg-white dark:bg-gray-900 text-gray-900 dark:text-white">
```

Comment:
the background is set on the html tag above to `bg-gray-100`. The duplication should be removed, and i'd prefer the background color remain the same as it was before